frontend/src/routes/(authenticated)/org/list/+page.svelte

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+  import { FilterBar } from '$lib/components/FilterBar';
+  import type { OrgListPageQuery } from '$lib/gql/types';
+  import t, { date, number } from '$lib/i18n';
+  import { Icon } from '$lib/icons';
+  import { Page } from '$lib/layout';
+  import { getSearchParams, queryParam } from '$lib/util/query-params';
+  import type { PageData } from './$types';
+  import type { OrgListSearchParams } from './+page';
+
+  export let data: PageData;
+  $: orgs = data.orgs;
+
+  const queryParams = getSearchParams<OrgListSearchParams>({
+    search: queryParam.string<string>(''),
+  });
+  const { queryParamValues, defaultQueryParamValues } = queryParams;
+
+  type OrgList = OrgListPageQuery['orgs']
+
+  type Column = 'name' | 'members' | 'created_at';
+  let sortColumn: Column = 'name';
+  type Dir = 'ascending' | 'descending';
+  let sortDir: Dir = 'ascending';
+
+  function swapSortDir(): void {
+    sortDir = sortDir === 'ascending' ? 'descending' : 'ascending';
+  }
+
+  function handleSortClick(clickedColumn: Column): void {
+    if (sortColumn === clickedColumn) {
+      swapSortDir();
+    } else {
+      sortColumn = clickedColumn;
+      sortDir = 'ascending';
+    }
+  }
+
+  function filterOrgs(orgs: OrgList, search: string): OrgList {
+    return orgs.filter((org) => org.name.toLowerCase().includes(search.toLowerCase()));
+  }
+
+  function sortOrgs(orgs: OrgList, sortColumn: Column, sortDir: Dir): OrgList {
+    const data = [... orgs];
+    let mult = sortDir === 'ascending' ? 1 : -1;
+    data.sort((a, b) => {
+      if (sortColumn === 'members') {
+        return (a.memberCount - b.memberCount) * mult;
+      } else if (sortColumn === 'name') {
+        const comp = a.name < b.name ? -1 : a.name > b.name ? 1 : 0;
+        return comp * mult;
+      } else if (sortColumn === 'created_at') {
+        const comp = a.createdDate < b.createdDate ? -1 : a.createdDate > b.createdDate ? 1 : 0;
+        return comp * mult;
+      }
+      return 0;
+    });
+    return data;
+  }
+
+  $: filteredOrgs = $orgs ? filterOrgs($orgs, $queryParamValues.search) : [];
+  $: displayOrgs = sortOrgs(filteredOrgs, sortColumn, sortDir);
+</script>
+
+<!--
+TODO:
+
+* Sort options: name, created date, # users
+* Paging
```

Comment:
I'm going to say paging is out of scope for this PR. We can add it later once we have enough orgs to page